gateway/1.4/modules/ROOT/pages/flex-gateway-k8-ingress-class.adoc

```diff
@@ -0,0 +1,61 @@
+= Configure Flex Gateway as an Ingress Controller in Local Mode
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+For Kubernetes deployments in Local Mode, Flex Gateway typically acts as an ingress controller that manages external access to your Kubernetes cluster.
```

Comment:
```suggestion
For Kubernetes deployments in Local Mode, Flex Gateway acts as an ingress controller that manages external access to your Kubernetes cluster.
```
Doesn't it always act as an ingress controller in Local Mode, not just typically?

Reply:
it does

Reply:
After talking with Sebastián, I think typically is still the right term.

Reply:
I'm confused. "Typically" implies there are _some_ instances when Flex in Local Mode does _not_ act like an ingress controller. It seems binary: either it acts like an ingress or it doesn't. I can see customers questioning the "typically". Perhaps I'm I misunderstanding something?